### ISSUE TITLE
ERROR   Internal error: have missed missed_tx_id.size()=?

### ISSUE THREAD

**Issue by aivve** [seredat/karbowanec#174]
Reported a bunch of such errors in daemon like following:
```
2022-Jan-09 17:18:25.182699 ERROR   Internal error: have missed missed_tx_id.size()=2
2022-Jan-09 17:18:25.182699 ERROR   for block id = b137b606b7a2ebad95d4a657cd6845e4c66e6361454f444892c7774425bdb7aa
2022-Jan-09 17:18:25.186697 ERROR   [213.136.89.252:32347 OUT] failed to handle request NOTIFY_REQUEST_GET_OBJECTS, dropping connection
```

**Comment by aivve** [comment]
Can be fixed by rescanning blockchain indexes, to trigger this remove all blockchain .dat and .bin files EXCEPT `blocks.dat` and `blockindexes.dat` and also leave intact `p2pstate.bin`

**Comment by aivve** [comment]
Could this be the side effect of BloDHa mining??

**Comment by eikoninaru** [comment]
karbowanec v. 1.9.6.1157-a7e767de


```
2022-Jan-20 21:09:42.012786 ERROR   Internal error: have missed missed_tx_id.size()=1
2022-Jan-20 21:09:42.012786 ERROR   for block id = 560c4f95915f10088ae9bdfdc6a2934272d620545e2ca2a5775f905ccff568ab
2022-Jan-20 21:09:42.013778 INFO    Probably some blockchain indexes or cache is corrupted. Perform rebuilding cache or just resync from scratch.
2022-Jan-20 21:09:42.014415 ERROR   [195.140.170.58:32347 OUT] failed to handle request NOTIFY_REQUEST_GET_OBJECTS, dropping connection
2022-Jan-20 21:10:22.375308 ERROR   Internal error: have missed missed_tx_id.size()=1
2022-Jan-20 21:10:22.375308 ERROR   for block id = 5d09e31374526fadbb03029da84f81a0d52394695ea0b711cb455ff04922c559
2022-Jan-20 21:10:22.375613 INFO    Probably some blockchain indexes or cache is corrupted. Perform rebuilding cache or just resync from scratch.
2022-Jan-20 21:10:22.375731 ERROR   [185.219.78.54:32347 OUT] failed to handle request NOTIFY_REQUEST_GET_OBJECTS, dropping connection

```
`rm -rf ~/.karbowanecd`

**Comment by aivve** [comment]
Attempt to fix it 6bc08f9e6c6b9088453880d204b6c9206ebdcc53

**Comment by aivve** [comment]
Seems to be gone after https://github.com/seredat/karbowanec/commit/6bc08f9e6c6b9088453880d204b6c9206ebdcc53
